### ISSUE TITLE
missing EPID provisioning error code

### ISSUE THREAD

**Issue by geogriff-signal** [intel/confidential-computing.sgx#300]
I am receiving an error message from aesmd during EPID Provisioning, which is reporting an EPID protocol error code without a corresponding definition in enum _se_protocol_response_status_t:

`[ADMIN]EPID Provisioning protocol error reported by Backend (9)`

Could you please explain error code 9 in the error reporting so I (and others in the future) have visibility as to what the problem is? I suspect it could be because I have an out-of-date microcode.

**Comment by llly** [comment]
You are right about failure. Error code 9 is SE_PRS_PROV_ATTEST_KEY_TCB_OUT_OF_DATE. The error code list will be updated.

**Comment by krzyszt1** [comment]
Error code 9 corresponds to `PVE_PROV_ATTEST_KEY_TCB_OUT_OF_DATE` (attestation key TCB out of date — i.e. an out-of-date TCB/microcode, as you suspected). This code is now defined and has a readable log string. Note that EPID-based provisioning has since been removed entirely (as of v2.28), so this path no longer exists in current releases. Closing as resolved.
